src/app/cli/src/graphql.ml

```diff
@@ -214,40 +214,51 @@ module Make (Commands : Coda_commands.Intf) = struct
         obj "Wallet" ~doc:"An account record according to the daemon"
           ~fields:(fun _ ->
             [ pubkey_field ~resolve:(fun _ account ->
-                  Stringable.public_key account.Account.Poly.public_key )
+                  (* Hack: Account.Poly.t is only parameterized over 'pk once
```

Comment:
There's some grammar mistakes here: But her is my revision of this:

```
                  (* Hack: Account.Poly.t is only parameterized over 'pk once
                   * and so in order for delegate to optional, we must also make
                   * account public_key optional even though it's always Some.
                   * In an attempt to avoid a large refactoring, and also avoid
                   * making a new record, we'll deal with a value_exn here and
                   * be sad. *)

```